### DISCUSSION TITLE
مناقشات

### DISCUSSION THREAD

**Discussion by assawalhy** [mobadarah/tajmeeaton#2] [General]
<div dir=rtl>

هنا يمكنك مناقشة
1. فكرة التجميعة
2. أفكار المشاريع المدرجة
3. كيف يمكنك أن نكون منتجي العلم والحضارة
4. إلخ

----

لتحويل اتجاه تصيير جيتهاب لملفات ال markdown ليكون من اليمين، قم بإضافة السطر التالي في بداية كتابتك:

</div>

```html
<div dir=rtl>
```

**Comment by assawalhy** [comment]
@linuxscout
@khaledhosny
@Sulaiman-SM
@Fahad-Alsaidi
@aboueleyes

**Reply by Sulaiman-SM** [reply]
@MuhammadSawalhy 
فكرة تجميعة ممتازة قد توفر للمطورين العرب الدعم جهدا أو مالا

قد نحتاج قناة أكثر فاعلية للنقاش التفصيلي مثل دسكورد

**Reply by assawalhy** [reply]
<div dir=rtl>

فكرة جيدة، سأقوم بذلك إن شاء الله.

</div>

**Reply by assawalhy** [reply]
@Sulaiman-SM https://discord.gg/Pu7Wjj6x

**Comment by assawalhy** [comment]
@KL13NT
@ejtaal
@OmarIthawi
@nasser
@hassandraga
@waseeld
@aboualiaa
@me00001
@alkhawarizm
@Yassine-Lafryhi

**Comment by assawalhy** [comment]
@Qutuf
@Muhammad-Altabba
@01walid
@mhsabbagh

**Comment by assawalhy** [comment]
@assem-ch

**Comment by assawalhy** [comment]
@drelhaj
@ahmedelq
@souhaib100
@AliOsm
@mohsenuss91

**Comment by assawalhy** [comment]
@malayyoub

**Reply by assawalhy** [reply]
@aiaf

**Reply by assawalhy** [reply]
@AhmedElTabarani

**Reply by assawalhy** [reply]
@DrWaleedAYousef

**Reply by assawalhy** [reply]
@mmahalwy
@tifa2UP
@hmzh-khn
@piraka9011
@abidlabs
@mmmoussa
@z3bi
@basememara
@meltuhamy
@ahmedre

**Reply by assawalhy** [reply]
@AbdelrahmanBayoumi
@YoussefMo7amed

**Reply by assawalhy** [reply]
@imAbdelhadi

**Comment by linuxscout** [comment]
الفكرة جميلة
ومجهود طيّب

**Comment by KL13NT** [comment]
<div dir="rtl">

هل من الممكن الدمج بين هذا الجهد و AIUX؟ ربما ايضاً اعادة النظر في بعض التقنيات المدرجة في الصفحة الرئيسية؟

**Reply by assawalhy** [reply]
<div dir=rtl>

وكيف يكون الدمج؟ أظن أن نعمل organization على جيتهاب يتضمن العملين، هل هذا ما تقصد؟

</div>

**Reply by assawalhy** [reply]
<div dir=rtl>

القائمة التي جمعتها تحتوى على أعمال بسيطة وأخرى أعمال ضخمة ومجهود بُذل، أعددتها على عجلة، ولا يزال هناك ما يجب تضمينة.

</div>

**Reply by assawalhy** [reply]
<div dir=rtl>

أنت الآن تستطيع التعديل: https://www.notion.so/scicave/3b693f2fe5ef4a08824aa7fd5fcd62fb

<div align=center>

![image](https://user-images.githubusercontent.com/42011920/123415129-57d71b80-d5a4-11eb-9b08-b6e944ed666f.png)

</div>

</div>

**Reply by KL13NT** [reply]
<div dir="rtl">

> 
> 
> وكيف يكون الدمج؟ أظن أن نعمل organization على جيتهاب يتضمن العملين، هل هذا ما تقصد؟

بالضبط. الهدف من AIUX هو توحيد الجهود العربية وغير العربية لتحسين دعم اللغة العربية وتحسين تجربة المستخدم الناطق باللغة العربية. وبالتالي قد يندرج هذا الجهد كجزء من AIUX.

**Reply by assawalhy** [reply]
<div dir=rtl>

تم دعوتك ل @mobadarah كمشرف عليها معي. سأنقل هذا المستودع هناك إن شاء الله.

</div>

**Reply by assawalhy** [reply]
??

**Comment by AhmedElTabarani** [comment]
<div dir=rtl>

@MuhammadSawalhy 

اظن في مشروع مشابه له 
هناك تجميعة كبيرة به

https://github.com/AqeelAT/arabic-apis

**Reply by assawalhy** [reply]
<div dir=rtl>

مضاف مسبقا في قسم ["تقنية -عام"](https://github.com/mobadarah/tajmeeaton#%D8%AA%D9%82%D9%86%D9%8A%D8%A9---%D8%B9%D8%A7%D9%85):

<div align=center>

![image](https://user-images.githubusercontent.com/42011920/123473212-c6899880-d5e7-11eb-86a0-f10d34667085.png)

</div>

</div>

**Reply by assawalhy** [reply]
<div dir=rtl>

ربما أقرب المشاريع لهذه التجميعة هو مشروع وليد، [01walid/awesome-arabic](https://github.com/01walid/awesome-arabic).

</div>

**Reply by assawalhy** [reply]
<div dir=rtl>

@01walid
هل أضيفك كمشرف مع في @mobadarah وأتيح لك التعديل في صقحة نوشن ونوحد الجهود؟

</div>

**Reply by Mohamed7AbdELhalim** [reply]
> مضاف مسبقا في قسم ["تقنية -عام"](https://github.com/mobadarah/tajmeeaton#%D8%AA%D9%82%D9%86%D9%8A%D8%A9---%D8%B9%D8%A7%D9%85):
> 
> ![image](https://user-images.githubusercontent.com/42011920/123473212-c6899880-d5e7-11eb-86a0-f10d34667085.png)



> مضاف مسبقا في قسم ["تقنية -عام"](https://github.com/mobadarah/tajmeeaton#%D8%AA%D9%82%D9%86%D9%8A%D8%A9---%D8%B9%D8%A7%D9%85):
> 
> ![image](https://user-images.githubusercontent.com/42011920/123473212-c6899880-d5e7-11eb-86a0-f10d34667085.png)

ما هي هذه ؟😅

**Comment by assawalhy** [comment]
![image](https://user-images.githubusercontent.com/42011920/129886568-a6b582f7-3cb4-4639-a77c-99a314d0e7d1.png)

**Reply by assawalhy** [reply]
أرى أن نركز جهودنا في المشاريع مفتوحة المصدر على مشاريع ستقدم نقلة كبيرة للغة العربية في حياتنا المعاصرة وتعريب العلوم، وباب الاقتراحات مفتوح لمن يريد أن يقترح مشاريع جديدة أو من أراد أن يناقش هذه المشاريع التي رأي  أهميتها.

**Reply by Mohamed7AbdELhalim** [reply]
يمكن أن تجد هذا في [لغة ألف البرمجية بالعربي](https://aliflang.org/) تم تنزيل النسخة الأولية اليوم يمكنك الإطلاع عليها.

1. لكن المشكلة هي 5 فقط من يقوم بحل المشاكل والتطوير فيها.
2.  أنه يجب ان تكون على معرفة بلغة سي ++ للتطوير معهم.
3. أن كان لديك مهارات أخر أقترحه عليهم ربما يحتاجون شيء ما.
4.  ...

**Comment by Mohamed7AbdELhalim** [comment]
السلام عليكم
### لماذا لا يتم نشر محتوى الروابط التي لا تعمل على موقع QUORA خاصة في مجتمع اللغة العربية أو على الفيسبوك في إحدى المجموعات البرمجية الكبيرة؟
 **قام بالفعل أحد مبرمجي لغة world بالعربي وأعلن عنها في مجموعة تحتوي على عدد كبير من المبرمجين (على الفيسبوك).**
 
### موضوع آخر (أفكر فيه طوال الوقت): 

- [ ]  أنا لدي فكرة في موضوع آخر لكن المشكلة أنني ليس لدي خبرة كافي لتصميمها أو امتلاك database للمستخدمين لحفظ أعملهم بالإضافة إلى تعدد ميزات المشروع فهو موقع إلكتروني نتعدد الاستخدامات ولا أركز على كونه وسائل تواصل اجتماعي أكثر من كونه مكان لنشر المعرفة من خلال ميزات متعددة متشابه إلى حدا ما مع مواقع أخرى. طوال الوقت أدون أفكار جديدة لكن مشكلة الدراسة وعدم امتلاك خبرة برمجية تحبط الشاب العربي (أنا) وحتى أن عرضتها على عربي آخر سوف يفكر بالمال قبل المنفعة منها. 

- وأنا أعرض فكرتي عليكم وأرجوا أن تخبروني بملاحظاتكم.

**Reply by assawalhy** [reply]
فكرتك غير واضحة

**Comment by hashimtree** [comment]
مشروع ممتاز جدا ، يفضل ان يشمل الشرح المبسط

**Reply by Mohamed7AbdELhalim** [reply]
ما هي اللغات التي يمكن استخدمها في هذا الموقع؟

**Comment by Mohamed7AbdELhalim** [comment]
طيب بالنسبة لموضوع الجداول انا هنا:
عملت fork
https://github.com/Mohamed7AbdELhalim/ai-in-a-week/blob/main/AI_Summaries%2F1%7C%20_%20.md  ليه انا مش عارف اخلي المجلد على اليمين


> <div dir=rtl>
> 
> هنا يمكنك مناقشة
> 1. فكرة التجميعة
> 2. أفكار المشاريع المدرجة
> 3. كيف يمكنك أن نكون منتجي العلم والحضارة
> 4. إلخ
> 
> ----
> 
> لتحويل اتجاه تصيير جيتهاب لملفات ال markdown ليكون من اليمين، قم بإضافة السطر التالي في بداية كتابتك:
> 
> </div>
> 
> ```html
> <div dir=rtl>
> ```

**Reply by assawalhy** [reply]
لا أفهم بالضبط استفسارك
